rpcs3/Emu/Cell/Modules/cellSysutilAvc.cpp

```diff
@@ -21,45 +47,45 @@ error_code cellSysutilAvcCancelJoinRequest()
 	return CELL_OK;
 }
 
-error_code cellSysutilAvcEnumPlayers()
+error_code cellSysutilAvcEnumPlayers(vm::pptr<SceNpId> players_id, vm::ptr<s32> players_num)
```

Comment:
vm::ptr<SceNpId> players_id, it's not a pointer to pointers, it's just a continuous array of SceNpId.